src/applications/disability-benefits/526EZ/tests/components/FormStartControls.unit.spec.jsx

```diff
@@ -47,7 +47,7 @@ describe('526 <FormStartControls>', () => {
         }}/>
     );
 
-    expect(tree.find('.usa-alert').text()).to.contain('To apply for a disability increase, you’ll need to verify your account');
+    expect(tree.find('.usa-alert').text()).to.contain('If you have a premium DS Logon or My HealtheVet account');
```

Comment:
Do the changes in DisabilityGate not produce any test errors? If that's the case, do we need to add any tests for those changes?